nsxt/resource_nsxt_policy_ipsec_vpn_session_test.go

```diff
@@ -485,6 +598,7 @@ func testAccNSXPolicyIPSecVpnSessionImporterGetID(s *terraform.State) (string, e
 }
 
 func testAccNsxtPolicyIPSecVpnSessionPreConditionTemplate(isT0 bool) string {
```

Comment:
Tests using certificate should only run if testAccEnvDefined(t, "NSXT_TEST_CERTIFICATE_NAME"), otherwise looks like certificate data source will fail

Reply:
done